File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/DynamicReadaheadConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class DynamicReadaheadConfig{

	@SerializedName("readAheadGrowthRateMs")
	private int readAheadGrowthRateMs;

	@SerializedName("maxReadAheadMediaTimeMs")
	private int maxReadAheadMediaTimeMs;

	@SerializedName("minReadAheadMediaTimeMs")
	private int minReadAheadMediaTimeMs;

	public void setReadAheadGrowthRateMs(int readAheadGrowthRateMs){
		this.readAheadGrowthRateMs = readAheadGrowthRateMs;
	}

	public int getReadAheadGrowthRateMs(){
		return readAheadGrowthRateMs;
	}

	public void setMaxReadAheadMediaTimeMs(int maxReadAheadMediaTimeMs){
		this.maxReadAheadMediaTimeMs = maxReadAheadMediaTimeMs;
	}

	public int getMaxReadAheadMediaTimeMs(){
		return maxReadAheadMediaTimeMs;
	}

	public void setMinReadAheadMediaTimeMs(int minReadAheadMediaTimeMs){
		this.minReadAheadMediaTimeMs = minReadAheadMediaTimeMs;
	}

	public int getMinReadAheadMediaTimeMs(){
		return minReadAheadMediaTimeMs;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/DynamicReadaheadConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class DynamicReadaheadConfig : Parcelable, Serializable {
    @SerializedName("readAheadGrowthRateMs")
    var readAheadGrowthRateMs = 0

    @SerializedName("maxReadAheadMediaTimeMs")
    var maxReadAheadMediaTimeMs = 0

    @SerializedName("minReadAheadMediaTimeMs")
    var minReadAheadMediaTimeMs = 0
    override fun toString(): String {
        return "DynamicReadaheadConfig{" +
                "readAheadGrowthRateMs = '" + readAheadGrowthRateMs + '\'' +
                ",maxReadAheadMediaTimeMs = '" + maxReadAheadMediaTimeMs + '\'' +
                ",minReadAheadMediaTimeMs = '" + minReadAheadMediaTimeMs + '\'' +
                "}"
    }
}